tests/components/template/test_alarm_control_panel.py

```diff
@@ -0,0 +1,645 @@
+"""The tests for the Template alarm control panel platform."""
+import logging
+
+from homeassistant import setup
+from homeassistant.components.alarm_control_panel import (
+    DOMAIN,
+    SERVICE_ALARM_ARM_AWAY,
+    SERVICE_ALARM_ARM_HOME,
+    SERVICE_ALARM_ARM_NIGHT,
+    SERVICE_ALARM_DISARM,
+)
+from homeassistant.const import (
+    STATE_ALARM_ARMED_AWAY,
+    STATE_ALARM_ARMED_HOME,
+    STATE_ALARM_ARMED_NIGHT,
+    STATE_ALARM_DISARMED,
+)
+from homeassistant.core import callback
+
+from tests.common import assert_setup_component, get_test_home_assistant
+
+_LOGGER = logging.getLogger(__name__)
+
+
+class TestTemplateAlarmControlPanel:
```

Comment:
New tests must be async standalone pytest test functions. See eg:
https://github.com/home-assistant/home-assistant/blob/f400b778371f032e26e3192870830719f16d96a2/tests/components/manual/test_alarm_control_panel.py#L85-L121

Reply:
Done :).

Tests updated to use the new format.